docs/user/security/access-agreement.asciidoc

```diff
@@ -0,0 +1,27 @@
+[role="xpack"]
```

Comment:
__note:__ didn't find any existing suitable section for this feature, so create a new one.